<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>627</x>
      <y>380</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/pic/HAIBARA 2.jpg);
border-image: url(:/new/prefix1/pic/HAIBARA1.jpg);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_new">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/pic/HAIBARA 2.jpg);
border-image: url(:/new/prefix1/pic/HAIBARA1.jpg);</string>
    </property>
    <property name="text">
     <string>HAIBARA</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>961</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="separator"/>
    <addaction name="CreateFile"/>
    <addaction name="CreateFolder"/>
    <addaction name="OpenFile"/>
    <addaction name="OpenFolder"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="CreateFile">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="CreateFolder">
   <property name="text">
    <string>新建文件夹
</string>
   </property>
  </action>
  <action name="OpenFile">
   <property name="text">
    <string>打开
</string>
   </property>
  </action>
  <action name="OpenFolder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
